controllers/grid/settings/category/form/CategoryForm.inc.php

```diff
@@ -211,21 +211,23 @@ public function fetch($request, $template = null, $display = false)
         $templateMgr->assign('sortOptions', Repo::submission()->getSortSelectOptions());
 
         // Sub Editors
-        $usersIterator = Services::get('user')->getMany([
-            'contextId' => $context->getId(),
-            'roleIds' => Role::ROLE_ID_SUB_EDITOR,
-        ]);
+        $usersIterator = Repo::user()->getMany(
+            Repo::user()->getCollector()
+                ->filterByContextIds([$context->getId()])
+                ->filterByRoleIds([Role::ROLE_ID_SUB_EDITOR])
+        );
         $availableSubeditors = [];
         foreach ($usersIterator as $user) {
-            $availableSubeditors[(int) $user->getId()] = $user->getFullName();
+            $availableSubeditors[$user->getId()] = $user->getFullName();
         }
         $assignedToCategory = [];
         if ($this->getCategoryId()) {
-            $assignedToCategory = Services::get('user')->getIds([
-                'contextId' => $context->getId(),
-                'roleIds' => Role::ROLE_ID_SUB_EDITOR,
-                'assignedToCategory' => (int) $this->getCategoryId(),
-            ]);
+            $assignedToCategory = iterator_to_array(Repo::user()->getIds(
+                Repo::user()->getCollector()
+                    ->filterByContextIds([$context->getId()])
+                    ->filterByRoleIds([Role::ROLE_ID_SUB_EDITOR])
+                    ->filterByAssignedCategoryIds([$this->getCategoryId()])
+            ));
```

Comment:
This probably works but since we're no longer returning the DAOResultIterator it may be better to use the Collection's `->toArray()` method.

Reply:
Done!